realm-jni/build.gradle

```diff
@@ -87,16 +87,39 @@ if (project.forceDownloadCore instanceof String) {
     ext.forceDownloadCore = Boolean.parseBoolean(project.forceDownloadCore)
 }
 
-task checkProperties(group: 'check', description: 'Check the user provided gradle.properties') << {
-    if (!project.ext['ndk.dir']) {
-        throw new GradleException('The ndkDir property in the gradle.properties file is not set.')
-    } else if (project.ext['ndk.dir'].endsWith('/')) {
-        throw new GradleException('The path provided in the ndkProperty in the gradle.properties ends with /')
-    } else if (!file(project.ext['ndk.dir']).directory) {
-        throw new GradleException('The path provided in the ndkDir property in the gradle.properties is not a folder.')
-    } else if (!file(new File(project.ext['ndk.dir'], 'RELEASE.TXT')).file) {
-        throw new GradleException('The path provided in the ndkDir property in the gradle.properties does not seem to be an Android NDK.')
+def getNdkDir() {
+    def ndkDir = null
+    if (project.ext.properties.containsKey('ndk.dir')) {
```

Comment:
Thanks @zaki50 for your review. It's 2AM KST and JST :) I am currently rebuild Realm to test `project.hasProperty`. I then will upload the new commit.